clients/repo_client.go

```diff
@@ -38,6 +45,8 @@ type RepoClient interface {
 	LocalPath() (string, error)
 	GetFileContent(filename string) ([]byte, error)
 	GetBranch(branch string) (*BranchRef, error)
+	GetCommitSHA() string
```

Comment:
I don't think this is a property of the repo client, but rather the repo right?

Reply:
Yeah but repos aren't accessible in `checks`